doc/admin-guide/files/storage.config.en.rst

```diff
@@ -184,3 +184,10 @@ following rules are stored in :manpage:`devfs.conf(5)`::
 
    # Assign /dev/ada1 and /dev/ada2 to the tserver user
    own    ada[12]  tserver:tserver
+
+Advanced
+--------
+
+When ``runroot`` generated from ``traffic_layout`` is used, :file:`storage.config` should be updated.
```

Comment:
"Because relative paths in :file`storage.config` are relative to the base prefix, when using ``runroot`` it may be necessary to adjust such paths in :file:`storage.config`. Despite the name, the ``cachedir`` value is not used for this file."